### ISSUE TITLE
Soonaverse requesting undiscounted payment

### ISSUE THREAD

**Issue by SeanTMartin** [soonaverse/soonaverse-dao#1041]
User has set up a discount in their collection, but when they try to purchase one, it only appears discounted, but trying to pay the discounted price fails. Using the tanglepay button results in the undiscounted amount being auto-filled.

![1-1](https://user-images.githubusercontent.com/38050901/217638723-a040cfe9-1775-4f4b-8e62-577e751a6db7.PNG)

Affected collection:
https://soonaverse.com/space/0x4dcfd426d4f47019fce9502bf484b51b19f84822/overview

**Comment by adamunchained** [comment]
I'm unable to replicate this. Could you check the exact steps to reproduce? Are you able to replicate this?

**Comment by SeanTMartin** [comment]
I'm not personally able to replicate this. Seeing if we can get any insight from the user.

**Comment by SeanTMartin** [comment]
From the user:

"I have found a workaround how the error does not occur anymore. I had entered one discount in my collection. When I entered a second one, it did not work. Now I have found that ONLY the FIRST discount works. I had entered the data from the second discount (100% discount) in the first place and it worked. So the problem is with the second discount and possibly any subsequent discount."
